packages/terra-i18n/README.md

```diff
@@ -12,6 +12,7 @@ The terra-i18n component provides the internationalization to the React componen
 
 ## Getting Started
 
+At most time, you don't need to install this package directly. Please follow [this internationalization section](../../README.md#internationalizationi18n) to set up. `Base` will install this package for you.
```

Comment:
I think this would be a better rephrasing of the sentence

    Note: If you are using `Base`, you do not need to install this component, as `Base` will include this module for you.